tests/e2e/scenarios/ha/test.yml

```diff
@@ -0,0 +1,237 @@
+- name: Skupper HA Test
+  hosts: all
+  connection: local
+
+  vars:
+    kubeconfig: kubeconfig
+
+  tasks:
+    - name: "[ HATest ] - Main HA test block"
+      block:
+        - ansible.builtin.debug:
+            msg: "[ HATest ] - Main HA test Playbook"
+
+        - name: "[ HATest - Setup ] Generating the namespaces"
+          ansible.builtin.include_role:
+            name: e2e.tests.generate_namespaces
+
+        - name: "[ HATest - Setup ] Creating Skupper resources on east_ha namespace"
+          skupper.v2.resource:
+            kubeconfig: "{{ kubeconfig }}"
+            path: "{{ item }}"
+            namespace: "{{ namespace_prefix }}-{{ namespace_name }}"
+          with_items:
+            - "{{ playbook_dir }}/resources/east_ha/backend.yaml"
+            - "{{ playbook_dir }}/resources/east_ha/site.yaml"
+            - "{{ playbook_dir }}/resources/east_ha/connector.yaml"
+          when:
+            - "'east' in inventory_hostname"
+
+        - name: "[ HATest - Setup ] Creating Skupper resources on west_ha namespace"
+          skupper.v2.resource:
+            kubeconfig: "{{ kubeconfig }}"
+            path: "{{ item }}"
+            namespace: "{{ namespace_prefix }}-{{ namespace_name }}"
+          with_items:
+            - "{{ playbook_dir }}/resources/west_ha/site.yaml"
+            - "{{ playbook_dir }}/resources/west_ha/listener.yaml"
+          when:
+            - "'west' in inventory_hostname"
+
+        - name: "[ HATest - Setup ] Issue a Skupper access token from west_ha namespace"
+          skupper.v2.token:
+            kubeconfig: "{{ kubeconfig }}"
+            namespace: "{{ namespace_prefix }}-{{ namespace_name }}"
+            name: west-ha-grant
+            redemptions_allowed: 1
+          register: west
+          when:
+            - "'west' in inventory_hostname"
+
+        - name: "[ HATest - Setup ] Apply token to east site"
+          skupper.v2.resource:
+            kubeconfig: "{{ kubeconfig }}"
+            def: "{{ hostvars['west_ha']['west']['token'] }}"
+            namespace: "{{ namespace_prefix }}-{{ namespace_name }}"
+          when:
+            - "'east' in inventory_hostname"
+
+        - name: "[ HATest - Setup ] Check the link status"
+          shell: kubectl get links -n  {{ namespace_prefix }}-{{ namespace_name }} --kubeconfig {{ kubeconfig }}
+          register: links_status
+          when:
+            - "'east' in inventory_hostname"
+
+        - name: "[ HATest - Setup ] Display the output of wait"
```

Comment:
Is this task misplaced? Should it be below the following one?
Or should the "Wait for links to get in Ready state" be executed earlier?